app/views/layouts/finder_layout.html.erb

```diff
@@ -16,7 +16,7 @@
       render_component_stylesheets
     %>
     <%= javascript_include_tag 'test-dependencies.js' if Rails.env.test? %>
```

Comment:
I wonder if we should also add type="module" here as well. It appears that test-dependencies.js is used to pull in [JS used by static to replicate the approach used on dev/production](https://github.com/alphagov/frontend/blob/main/app/assets/javascripts/test-dependencies.js#L1-L2), not 100% sure if this would actually make any difference to the tests, but would just align the approach used (to be used) more closely.